@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf

```diff
@@ -84,6 +84,8 @@ if(fza_ah64_sfmPlusStabilatorEnabled == STABILATOR_MODE_ALWAYSENABLED
     [_heli, _deltaTime, _dryAirDensity] call fza_sfmplus_fnc_aeroStabilator;
 };
 
+[_heli] call fza_sfmplus_fnc_probes;
```

Comment:
You're not calling blades? Also please can we ensure
Yes we still don't call in MP